okapi-core/src/main/java/org/folio/okapi/managers/TenantManager.java

```diff
@@ -544,7 +545,7 @@ private void fireTimer(Tenant tenant, ModuleDescriptor md, RoutingEntry re, Stri
   private Future<Void> invokePermissionsForModule(Tenant tenant, ModuleDescriptor mdTo,
                                                   ModuleDescriptor permsModule, ProxyContext pc) {
 
-    pc.debug("Loading permissions for " + mdTo.getName()
+    logger.debug("Loading permissions for {}", () -> mdTo.getName()
```

Comment:
Would this work `logger.debug("Loading permissions for {}", mdTo.getName()`?

Reply:
Yes. The question is what is fastest. In this particular case, since the getName is a fast operation, using a lambda is probably overkill.

Reply:
I'll change it to normal.. By the way.. the following line is a mess.